docs/quickstart.rst

```diff
@@ -23,6 +23,16 @@ Fairlearn is also available on
 For checking out the latest version in our repository check out our
 :ref:`advanced_install`.
 
+.. note::
+
+    The Fairlearn API is still evolving, so example code in 
+    this documentation may not work with every version of Fairlearn.
+    Please use the version selector to get to the instructions for
```

Comment:
nit: I'd be very specific and say it's in the "top left corner".

I also found that it's apparently possible to update tags: https://git-scm.com/book/en/v2/Git-Basics-Tagging

Given that we're soon on a new version I don't know if it's worth the effort. We should just make sure that the default website points to v0.5.0 once it's out.

Reply:
Not sure what you mean about tagging (although they are certainly mutable in Git). I did consider directing people to the location of the version picker, but that could change in future....